src/main/java/org/broadinstitute/hellbender/tools/spark/sv/RunSGAViaProcessBuilderOnSpark.java

```diff
@@ -511,6 +511,22 @@ public String toString(){
             public String toString(){
                 return id;
             }
+
+            @Override
+            public boolean equals(final Object o) {
+                if (this == o) return true;
+                if (o == null || getClass() != o.getClass()) return false;
```

Comment:
This is fine with me, but you could consider:
`if ( !(o instanceof ContigID) ) return false;`